### ISSUE TITLE
Pretrain the model on UCF101

### ISSUE THREAD

**Issue by wangll1212** [amazon-research/video-contrastive-learning#11]
Hi, I pretrained the model on UCF101, and the  linear evaluation on UCF101 is 74.0946%. 

![1](https://user-images.githubusercontent.com/71969945/173597775-2230691d-dc28-45e8-8b23-832779bafba9.png)

But the result of the pretrained model you provided is 59.8731%. 

![2](https://user-images.githubusercontent.com/71969945/173597843-c3e278ec-460d-4737-8b38-97949608e4c7.png)

I just modified the code of the data loading part, and the other settings have not been changed.
 Do you have any changes to the settings when you pretrain the model on UCF101?

**Comment by bryanyzhu** [comment]
@KuangHaofei Can you help to look into this? Thank you.

**Comment by KuangHaofei** [comment]
Sorry! I am trying to recall the details about the UCF101 pretraining.

Because the experiment of UCF101 pretraining is used for a fair comparison with other methods for video retrieval, I didn't pay much attention to it such that I could not find the logs related to the pretraining now. So sorry for that!

Whatever got a higher performance is always good news! But I have a question here, which split of UCF101 dataset is used for your pretraining?

**Comment by wangll1212** [comment]
I used the split_1 for pretraining the model.

**Comment by KuangHaofei** [comment]
OK, it should be the same as us. Did you prepare the UCF101 dataset as shown in the [PREPARE_DATA.md](https://github.com/amazon-research/video-contrastive-learning/blob/main/PREPARE_DATA.md)?

If you follow the tutorials to get the dataset and use split_1, you would use the same data with us. I cannot recall the specific pretraining setups, sorry for that! But I think you can trust your results if you get higher performance, and go ahead with your project.

We will try to redo the experiment! And you can keep the issue open until we make sure there are no problems! 

Thanks!

**Comment by wangll1212** [comment]
Yes, I prepared the UCF101 dataset as shown in the PREPARE_DATA.md. 
Thank you very much for your reply and look forward to your results！
